e2e/atlas/live_migrations_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 MongoDB Inc
```

Comment:
```suggestion
// Copyright 2021 MongoDB Inc
```

Reply:
G*D D**N it, I keep checking those and missing one when doing a copy/pasta :D